irohad/model/block.hpp

```diff
@@ -18,12 +18,13 @@
 #ifndef IROHA_BLOCK_HPP
 #define IROHA_BLOCK_HPP
 
-#include <common/types.hpp>
-#include <model/proposal.hpp>
-#include <model/signature.hpp>
-#include <model/transaction.hpp>
 #include <vector>
 
+#include "crypto/hash_types.hpp"
```

Comment:
I am sad about supporting legacy code. This is of course not an issue for your pr.